.travis.yml

```diff
@@ -24,6 +24,7 @@ env:
         - TOXENV='test'
         - TOXARGS='-v'
         - TOXPOSARGS=''
+        - CONDA_DEPENDENCIES="pandas pytables"
```

Comment:
We don't use conda anymore for the CI so this should be removed

Reply:
how do I install pandas?